src/Dialect/ONNX/ONNXOps/NN/Normalization.cpp

```diff
@@ -149,6 +149,21 @@ LogicalResult ONNXInstanceNormalizationOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// GroupNormalizationV18
```

Comment:
@AlexandreEichenberger Is this what you were thinking of? I added a print because if I add `emitWarning` it seems like all opset 18 tests fail and I figure we can still enable support for the meantime. I am fine with removing support in the near future but was not sure if any model still uses GroupNorm Opset 18.